integrations/common/vars.go

```diff
@@ -28,6 +30,24 @@ type OAuthData struct {
 	TokenType    string `var:"oauth_token_type"`
 }
 
+// CheckOAuthToken returns a warning status if OAuthAccessTokenVar is missing in sdktypes.Vars; otherwise, it returns OK.
+// It depends on sdktypes.Vars being preloaded with OAuthAccessTokenVar, which isn't validated.
```

Comment:
https://go.dev/doc/comment#doclinks + cosmetic tweak (shorter 1st line)
```suggestion
// CheckOAuthToken returns a warning status if [OAuthAccessTokenVar] is missing in [sdktypes.Vars]; otherwise,
// it returns OK. It depends on [sdktypes.Vars] being preloaded with [OAuthAccessTokenVar], which isn't validated.
```